### PR TITLE
Fix build problems

### PR DESCRIPTION
**Pull request by AndrewJSchofield**
Fixing a couple of minor build problems that slipped through.

### CODE REVIEW

File: core/src/main/java/io/confluent/rest/ApplicationServer.java [hunk 1]
```diff
@@ -394,7 +394,8 @@ private void configureConnectors(SslContextFactory sslContextFactory) {
             new HttpConnectionFactory(httpConfiguration);
 
     // Default to supporting HTTP/2 for Java 11 and later
-    final boolean http2Enabled = isJava11Compatible() && config.getBoolean(RestConfig.HTTP2_ENABLED_CONFIG);
+    final boolean http2Enabled = isJava11Compatible()
+                              && config.getBoolean(RestConfig.HTTP2_ENABLED_CONFIG);
```
Comment (dimitarndimitrov) [inline core/src/main/java/io/confluent/rest/ApplicationServer.java:398]: nit: I think the indent here should be 2x the regular indent, or 4 spaces.

File: core/src/test/java/io/confluent/rest/Http2Test.java [hunk 2]
```diff
@@ -257,7 +259,9 @@ private int makeGetRequestHttp(String url, String clientKeystoreLocation, String
                                 String clientKeyPassword)
       throws Exception {
     log.debug("Making GET using HTTPS " + url);
-    HttpClient httpClient = new HttpClient(buildSslContextFactory());
+    HttpClient httpClient = new HttpClient(buildSslContextFactory(clientKeystoreLocation,
+                                                                  clientKeystorePassword,
+                                                                  clientKeyPassword));
```
Comment (dimitarndimitrov) [inline core/src/test/java/io/confluent/rest/Http2Test.java:264]: Ha, interesting - did this (and the similar code below) cause a test failure?

**Review by swist — APPROVED** [review]
Looks good to me! Thanks for taking care of it

**Review by dimitarndimitrov — APPROVED** [review]
+1.

Thanks for catching and addressing this really quick!

File: core/src/test/java/io/confluent/rest/Http2Test.java (hunk 2 above)
Comment (AndrewJSchofield) [reply core/src/test/java/io/confluent/rest/Http2Test.java:264]: The maven target I was using to check the tests didn't actually rebuild then, which was a surprise.

**Pull request merged**